esi_leap/api/controllers/v1/lease.py

```diff
@@ -234,6 +236,8 @@ def _lease_get_all_authorize_filters(cdict,
                                                cdict, cdict)
                     possible_filters['project_id'] = project_id
 
+        possible_filters['time_filter_type'] = constants.WITHIN_TIME_FILTER
```

Comment:
Ah, sorry, one last change - this value is always set for the possible_filters, which I think is fine, but I think you can then just set it when possible_filters is declared above.

Reply:
Ah okay, will do!